src/components/AppBars/AppBarBase/AppBarBase.stories.tsx

```diff
@@ -17,17 +17,21 @@ export default {
     },
 };
 
-export const Simple: Story<AppBarBaseProps> = (args) => {
+export const Basic: Story<AppBarBaseProps> = (args) => {
     return (
         <AppBarBase {...args}>
             <ToolbarBase>
                 <DrawerTriggerButton>
                     <MenuIcon />
                 </DrawerTriggerButton>
-                <TypographyBase variant="h6" noWrap>
-                    Simple App Bar Drawer
+                <TypographyBase variant="h6" component="div" sx={{ flexGrow: 1 }}>
```

Comment:
why do you want it to be a div? Why usse flexGrow?

Reply:
Actually, I don't know. I see it in the Basic App Bar in Mui document. You can see the image below.
Here is the link: https://mui.com/components/app-bar/
![image](https://user-images.githubusercontent.com/59761256/144744252-e75d27dc-cf37-43e9-bdb9-f3edb7f40342.png)